### DISCUSSION TITLE
Is this to risky?

### DISCUSSION THREAD

**Discussion by philhk** [Vedza/NitroSniperGo#349] [General]
So I created 100 alts now all with a different vpn and every single one of them with the browser cache cleared. Now I am wondering if it is save to enable it all accounts are on the same servers for giveaways and if I would put the delay of reacting to it to like 10 seconds would all accounts instantly react to it after 10 seconds or 1 account per 10 seconds? And another question would be wouldn't discord see that 100 accounts are logged in at the same place (ip address)? Can I use proxys for the bot in some way? And something weird was happening to me while creating those accounts every time I created one it said "something's going on here" idk if it was because of the vpn... And maybe someone has some advice for safer snipping? Thx in advance.

**Comment by Vedza** [comment]
Just use the nitro sniper, disable giveaways joiner and dm and you're good 👍 
You don't need more to get Nitro. And with 100 alts it should be petty fast.
Proxy would make it slower so it doesn't make sense.

**Reply by philhk** [reply]
Okay maybe you are right with the proxys but I think you didn't really understand what I meant all of those 100 alts are on the same servers (The servers do many discord nitro giveaways) and when a giveaway start all alts are reacting to the giveaway at the same time and that looks really suspicous.

**Reply by Vedza** [reply]
I understood what you meant, this is why I told you to disable giveaway joiner. I'm pretty sure the biggest parts of ban I saw were because of giveaways. So with with 100 alts, I'm pretty sure some alts are going to get ban, on the same server or not. And you don't need it to get Nitro with 100 alts ( even with 2 alts in my opinion ). So just disable it
